flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerServices.java

```diff
@@ -116,8 +116,15 @@ public static JobManagerServices fromConfiguration(
 		final String classLoaderResolveOrder =
 			config.getString(CoreOptions.CLASSLOADER_RESOLVE_ORDER);
 
+		final String alwaysParentFirstLoaderString =
+			config.getString(CoreOptions.ALWAYS_PARENT_FIRST_LOADER);
+		final String[] alwaysParentFirstLoaderPatterns = alwaysParentFirstLoaderString.split(",");
+
 		final BlobLibraryCacheManager libraryCacheManager =
-			new BlobLibraryCacheManager(blobServer, FlinkUserCodeClassLoaders.ResolveOrder.fromString(classLoaderResolveOrder));
+			new BlobLibraryCacheManager(
+				blobServer,
+				FlinkUserCodeClassLoaders.ResolveOrder.fromString(classLoaderResolveOrder),
```

Comment:
Does this fail with a very ugly enum exception when someone misspells the enum literal?

Reply:
This will fail with `IllegalArgumentException("Unknown resolve order: " + resolveOrder)`.

Reply:
Okay, that works.